nestor_api/api/api_routes/sample.py

```diff
@@ -0,0 +1,11 @@
+"""Define the sample route."""
+from flask import Blueprint, Response, jsonify
+
+
+def register_routes(api: Blueprint) -> None:
+    """Register the /sample route."""
+
+    @api.route("/sample", methods=["GET"])
```

Comment:
Do we really need this sample that will be removed quite soon anyway ?

Reply:
It's just to have non empty folder, but not mandatory, it is so that people don't define routes in `public_routes` which is only used for the heartbeat route.